### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by osterman**
## what
* Clean up README
* Fix LICENSE

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    packages:
+      - git
+      - make
+      - curl
+
+install:
+  - make init
+
+script:
+  - make terraform:install
```
Comment (aknysh) [inline .travis.yml]: use `/` instead of `:` in Make targets

File: docs/terraform.md [hunk 2]
```diff
@@ -5,16 +5,16 @@
 |------|-------------|:----:|:-----:|:-----:|
 | account_id | The account id of the parameter store you want to allow access to. If none supplied, it uses the current account id of the provider. | string | `` | no |
 | kms_key | The arn of the KMS key that you want to allow access to. If empty it uses a wildcard resource. `*` | string | `` | no |
```
Comment (aknysh) [inline docs/terraform.md]: `.` in wrong place at the end of the description

File: outputs.tf [hunk 3]
```diff
@@ -1,24 +1,24 @@
 output "read_parameter_store_policy" {
-  description = "A JSON policy document that only allows read access to the parameter store"
+  description = "A JSON policy document that only allows read access to the parameter store."
   value = "${data.aws_iam_policy_document.read_parameter_store.json}"
```
Comment (aknysh) [inline outputs.tf]: terraform fmt

**Review by aknysh — COMMENTED** [review]
inline comments

**Comment by Jamie-BitFlight** [comment]
Addressing now team.

**Comment by osterman** [comment]
@aknysh fixed

**Review by Jamie-BitFlight — APPROVED** [review]

**Pull request merged**